src/ttauri/geometry/axis_aligned_rectangle.hpp

```diff
@@ -418,7 +418,7 @@ class axis_aligned_rectangle {
      * @param rectangle The rectangle to fit inside the bounds.
      * @return A rectangle that fits inside the bounds
      */
-    [[nodiscard]] friend axis_aligned_rectangle
+    friend axis_aligned_rectangle
```

Comment:
you should never discard the result of fit.

Reply:
Re-added the `[[nodiscard]]`. This gives the following Clang error: https://godbolt.org/z/d546Pc